smartBugs.py

```diff
@@ -15,47 +17,51 @@
 from src.output_parser.SarifHolder import SarifHolder
 from time import time, localtime, strftime
 
+SMARTBUGS_FILE_PATH = os.path.dirname(os.path.realpath(__file__))
+DATASET_CONFIG_FILE_PATH = SMARTBUGS_FILE_PATH + '/config/dataset/dataset.yaml'
 
-cfg_dataset_path = os.path.abspath('config/dataset/dataset.yaml')
+cfg_dataset_path = os.path.abspath(DATASET_CONFIG_FILE_PATH)
 with open(cfg_dataset_path, 'r') as ymlfile:
     try:
         cfg_dataset = yaml.safe_load(ymlfile)
     except yaml.YAMLError as exc:
-        print(exc)
+        raise Exception('Dataset YAML not found.'
```

Comment:
It would be best to create the `SmartBugsException` and use it instead of the generic `Exception`.